kentik_api_library/setup.py

```diff
@@ -106,7 +106,7 @@ def run(self):
     long_description_content_type="text/markdown",
     url="https://github.com/kentik/community_sdk_python/tree/main/kentik_api_library",
     include_package_data=True,
-    install_requires=["python-http-client>=3.3.1", "requests>=2.25.0", "typing-extensions>=3.7.4.3"],
+    install_requires=["python-http-client>=3.3.1", "requests>=2.25.0", "typing-extensions>=3.7.4.3", "dacite>=1.6.0"],
```

Comment:
Shouldn't we add dacite to `requirements.txt`?
Also what subset of dependencies is listed in install_requires? All 3rd party libraries?

Reply:
Yes, we should add dacite to requirements.txt.
The install_requires should be used to specify what a project minimally needs to run correctly (more information can be found here: https://packaging.python.org/discussions/install-requires-vs-requirements/).